src/contentscript/core.ts

```diff
@@ -537,4 +537,9 @@ export default class Core {
   public connectedAccounts = new ConnectedAccounts()
 
   public fetch = proxyFetch
+
+  public async getPreferredConnectedAccountsNetwork() {
```

Comment:
Should dapplets know about it?

Reply:
Yes. To understand what the network type use in the login to work with Connected Accounts.